relay-event-schema/src/protocol/span/convert.rs

```diff
@@ -238,7 +238,41 @@ mod tests {
             profile_id: EventId(
                 a0aaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaab,
             ),
-            data: ~,
+            data: SpanData {
```

Comment:
Is this change expected?

Reply:
Yep, we know map `event.release` to `span.data.release` (same for `environment`).

Reply:
And that causes span data to exist?

Reply:
Yes, there was no field that was mapped to `span.data` before so `span.data` was an empty `Annotated` (`data: ~`).